tests/conftest.py

```diff
@@ -43,10 +43,19 @@ def pytest_collection_modifyitems(config: Config, items: list[Any]) -> None:
     # If e2e tests are requested, skips everything not marked as e2e
     skip_e2e = pytest.mark.skip(reason="only running non-e2e tests")
     skip_non_e2e = pytest.mark.skip(reason="only running e2e tests")
+    skip_e2e_environment = pytest.mark.skip(reason="only run this e2e test in a specific environment")
```

Comment:
this is the reason shown within a specific pytest run, rather than the reason the market exists, so maybe a small tweak

```suggestion
    skip_e2e_environment = pytest.mark.skip(reason="test is configured not to run in this e2e environment")
```